Js/script.js

```diff
@@ -0,0 +1,2 @@
+$('.alert').delay(3000).fadeOut('slow',function() { $(this).remove(); });
+console.log('test');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Zopiote/P5/pullRequest?prid=8313983)
```suggestion
console.log("test");
```